### ISSUE TITLE
Research market and create design docs for new project

### ISSUE THREAD

**Issue by giraffekey** [ArtemGr/bounty#11]
I am starting a new project called Ogre Drive which is going to function as an open source GUN/Sia-based alternative to Google Drive/OneDrive/etc.

This is going to be the first project to use Griffin in production. It has two objectives: challenge and compete with centralized, proprietary solutions to cloud storage, and provide real-world testing for Griffin's use cases. I expect this to help Griffin develop further than brainstorming and unit tests alone will.

This task is specifically for me. What I need to do is:
- Research similar applications
- Create UX/features documentation
- Plan out project tasks in Basecamp
- Create UI designs in Figma

**Comment by giraffekey** [comment]
I have created the template for the project here: https://github.com/OgreApps/drive

The UX/features documentation was done very thoroughly, I have a very good idea of what to do and how to implement it now.

For project tasks, I decided I'd rather use Github issues than Basecamp or Trello as I tend not to need kanban that much and it's more convenient to keep everything in the same place. Maybe will use Trello in the future but don't feel I need it now. In early development I prefer to just wing it.

For UI designs, I have so much reference material for what I want and such a detailed description in the plan documents that I feel I don't need Figma to formalize that; I can just formalize it with CSS. Might use Figma when designing the home page but app designs tend to be too simplistic to be worth the time and I'm good at formalizing things within my head.

Also I spent a lot of time on the Gulp setup, learning more about Scala, learning more about PWAs, etc, so I am a bit worn out from the setup phase and would like to move on.

So, I consider this task complete. Competitors have been thoroughly researched and UX docs have been created.

Going to get started with development now!

**Comment by giraffekey** [comment]
![image](https://user-images.githubusercontent.com/11844126/106339846-a62dfb80-624c-11eb-86e6-d376b1f8b074.png)

**Comment by giraffekey** [comment]
UI I created with CSS

**Comment by ArtemGr** [comment]
Neat!

Regarding the kanban, have you seen/used the GitHub project boards?

Some interesting tidbit(s) you've learned about the competitors?

**Comment by giraffekey** [comment]
Ah yeah I have seen the kanban boards I guess I just need to explore Github more.

Something interesting I noticed was they all placed a major focus on integrations. They also place limitations on storage and have inflexible pricing. Google Drive doesn't let you store more than 2TB I think. I also noticed Dropbox is very sucky lol, it has barely any features not sure why it's so popular.

I think open source projects can definitely beat proprietary ones when it comes to integrations and Sia has practically unlimited storage with very flexible pricing. I plan to only officially integrate with OS projects, so Taiga instead of Trello, Zulip instead of Slack, Mastodon instead of Twitter, etc. If I don't find something that meets my standards I might make it another Ogre app, for example I also plan on eventually creating alternatives to docs/sheets/slides.
